workbench/__init__.py

```diff
@@ -2,4 +2,4 @@
 Provide a djangoapp for XBlock development
 """
 
-__version__ = '0.11.0'
+__version__ = '1.0.0'
```

Comment:
If xblock-sdk were a truly supported tool, I would agree with bumping the major version, since dropping py38 support is a breaking change. However, I would describe our xblock-sdk support is as "provisional" at best. We don't manually test it in releases and I hear that several production blocks do not work in it. To indicate this, I would keep the major version at `0.` and bump the minor version instead.
```suggestion
__version__ = '0.12.0'
```

Reply:
make sense. Thanks for reviewing